### DISCUSSION TITLE
How can you use the `dotnet new` templates?

### DISCUSSION THREAD

**Discussion by ThaDaVos** [wixtoolset/issues#8122] [Questions]
There was an issue: https://github.com/wixtoolset/issues/issues/6207
And a corresponding pullrequest: https://github.com/wixtoolset/wix/pull/150

But I don't see any templates in my `dotnet new list` output - is it even there?

**Comment by robmen** [comment]
Never released. The overhead of maintaining them wasn't worth the effort.

**Comment by chrpai** [comment]
I could add this to IsWiX if there's great demand for this.  So far I only have one customer using VSCode instead of Visual Studio and he doesn't have any desire to create new projects but only to maintain the one that I created for him.

I've been thinking of a use case where I port IsWiX to MAUI so it'll run on a Mac. The idea being a developer creating an electron/node/go/python app for windows could create the project template, use iswix and vscode to author the installer and then build it via CI/CD pipeline.  If. WiX ever worked on MacOS then they could build it locally.

This is dreaming but then again these are exactly the sort of scenarios Microsoft has embraced in the past ten years. Not that any of us have their budget though.
